CHANGES.txt

```diff
@@ -18,6 +18,9 @@ Breaking Changes
 Changes
 =======
 
+ - Added support for hash sums (MD5, SHA1, SHA256). Please
+    see :ref:`sha1 <sha1>`.
```

Comment:
This reference here won't work as CHANGES.txt is not part of the generated html documentation.